### DISCUSSION TITLE
Extension for visual studio code

### DISCUSSION THREAD

**Discussion by black-ghost-off** [zephyrproject-rtos/zephyr#72736] [General]
Created a convenient plugin for vscode for zephyros development
https://github.com/mylonics/zephyr-ide

**Environment**

* Page: `develop/tools/index`
* Version: main
* SHA-1: 611d284fc6466b96943be9489364e1f9f34df73f

**Comment by koffes** [comment]
Nice! Do note that there is a large discussion on this topic here: https://github.com/zephyrproject-rtos/zephyr/issues/21119
